packages/core/src/notification.ts

```diff
@@ -0,0 +1,592 @@
+/**
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as json from './json'
+import { util } from './internal'
+
+interface NotificationPosition {
+  offset?: number
+  line?: number
+  column?: number
+}
+
+type UnknownGqlStatus = `${'01' | '02' | '03' | '50'}N42`
+
+const unknownGqlStatus: Record<string, { gql_status: UnknownGqlStatus, status_description: string }> = {
+  WARNING: {
+    gql_status: '01N42',
+    status_description: 'warn: warning - unknown warning'
+  },
+  NO_DATA: {
+    gql_status: '02N42',
+    status_description: 'note: no data - unknown subcondition'
+  },
+  INFORMATION: {
+    gql_status: '03N42',
+    status_description: 'info: informational - unknown notification'
+  },
+  ERROR: {
+    gql_status: '50N42',
+    status_description: 'error: general processing exception - unknown error'
+  }
+}
```

Comment:
```suggestion
  WARNING: {
    gql_status: '01N42',
    status_description: 'warn: unknown warning'
  },
  NO_DATA: {
    gql_status: '02N42',
    status_description: 'note: no data - unknown subcondition'
  },
  INFORMATION: {
    gql_status: '03N42',
    status_description: 'info: unknown notification'
  },
  ERROR: {
    gql_status: '50N42',
    status_description: 'error: general processing exception - unknown error'
  }
}
```
Please double-check. This is how I read section 4.9.2 of the standard anyway.

Reply:
The description is define das `${short severity}: ${condition} - ${subcondtion}`

for 03N42, condition is `informational` and subcondition is `unknown notification`, for example. See the codes spreadsheet.